jdaviz/app.py

```diff
@@ -1347,6 +1347,47 @@ def merge_func(spectral_region):  # noop
             elif isinstance(subset_state, MultiMaskSubsetState):
                 return self._get_multi_mask_subset_definition(subset_state)
 
+    def delete_subsets(self, subset_labels=None):
+        """
+        Delete all or specified subsets in app.
+
+        This method removes subsets based on the provided `subset_labels` (a
+        single subset label or multiple labels). If `subset_labels`
+        is None, all subsets will be removed.
+
+        Parameters
+        ----------
+        subset_labels : str or list of str or None
+            The label(s) of the subsets to delete. If None, all subsets will be
+            removed.
+
+        Returns
+        -------
+        None
```

Comment:
Returning None is implicit across Python, so there is no need to put this in the docstring.
```suggestion
```